deploy/K8s-onprem-MIG/Readme

```diff
@@ -0,0 +1,5 @@
+This repo provides the scripts for autoscaling Triton deployment with Kubernetes (on MIG) and load balacing using NGNIX Plus.
```

Comment:
This directory should follow existing naming conventions, so no caps. Also, why is this onprem specific. Also, why is it MIG specific. It seems that mig is only mentioned in one resource limit... but couldn't that instead just be a variable that users can set to deploy to many different types of resources. Certainly you can call out how this example can be used to deploy to mig but it doesn't need to be only mig.

The fundamental contribution seems to be that it autoscales based on some metrics and that it uses nginx. So can we name directory nginx_autoscale?

Reply:
I tried to rename it but seems I don't have the permission, could you please help rename it to hpa_autoscale, or hpa_autoscale_nginx? Thanks